git-gud.1

```diff
@@ -0,0 +1,68 @@
+.TH GIT-GUD 1 "19 Feb 2020" "version 0.0"
+.SH NAME
+git-gud \- get good at git with git-gud
+.SH SYNOPSIS
+.B git gud
+[option1] [option2]
+.SH DESCRIPTION
+.B git-gud
+teaches you how to use Git with fun interactive tutorials.
+Use the following command to create a Git repository and start the first level:
+.B git\ gud\ init
```

Comment:
From the previous review:

> I looked at the man page for git, and it seems that all the git commands where you write git init etc are actually all their own commands and all have their own man page. Would it make more sense to actually convert git gud's subcommands into actual subcommands so things like git-gud-load are actually their own unique command and not just an argument to the git-gud command? That seems to be what git does with all of it's commands, and I'd love to stay in line with what Git's doing.

Can you please respond to this?

I'm thinking visually, this would look like `git-gud-init(1)` and I would have to create a new command entrypoint that would directly correspond with the `git-gud-init` command rather than a processed subcommand

Reply:
Okay, that sounds good. I think that can be done after the PR though, as these would be separate man pages and this one is just for git-gud for now.